### PR TITLE
support eth_getLogs

### PR DESCRIPTION
**Pull request by jmank88**
This PR proposes a minimal change to support `eth_getLogs`.

### CODE REVIEW

File: lib/src/core/client.dart [hunk 1]
```diff
@@ -252,6 +252,14 @@ class Web3Client {
         'eth_getCode', [address.hex, _getBlockParam(atBlock)]).then(hexToBytes);
   }
 
+  Future<List<FilterEvent>> eth_getLogs(FilterOptions options) {
```
Comment (jmank88) [inline lib/src/core/client.dart:255]:
```suggestion
  Future<List<FilterEvent>> getLogs(FilterOptions options) {
```

File: lib/src/core/client.dart [hunk 2]
```diff
@@ -252,6 +252,14 @@ class Web3Client {
         'eth_getCode', [address.hex, _getBlockParam(atBlock)]).then(hexToBytes);
   }
 
+  Future<List<FilterEvent>> getLogs(FilterOptions options) {
+    final filter = _EventFilter(options);
+    return _makeRPCCall<List<dynamic>>('eth_getLogs', [
+      filter._createParamsObject(true)
+    ]).then(
+        (logs) => logs != null ? logs.map(filter.parseChanges).toList() : null);
```
Comment (simolus3) [inline lib/src/core/client.dart:260]: I would convert this method to a block body for nicer formatting.

**Comment by jmank88** [comment]
Updated!

**Comment by simolus3** [comment]
Thanks! I'll release this as `2.0.0-dev.3` as soon as Travis runs through.

**Pull request merged**